packages/fastnetmon/PKGBUILD

```diff
@@ -0,0 +1,28 @@
+# Maintainer: h0bbl3s <david.frisbee@gmail.com>
+
+pkgname="fastnetmon"
+pkgver=1.1.3.c74470e92
+pkgrel=1
+pkgdesc="ddos detection and network monitor using pf_ring"
+arch=(x86_64)
+url="www.fastnetmon.com"
+license=('GPL')
+groups=('blackarch' 'blackarch-networking')
+depends=('ndpi' 'log4cpp' 'libpfring' 'pfring-kernel' 'luajit' 'boost' 'boost-libs')
+makedepends=('pfring-kernel' 'libpfring' 'cmake' 'linux-headers')
+provides=('fastnetmon')
+source=("git+https://github.com/h0bbl3s/fastnetmon.git")
+md5sums=('SKIP')
+
```

Comment:
When pulling from git, you need a pkgver function that is how we keep track of when to update those packages.
Check: https://wiki.archlinux.org/index.php/VCS_package_guidelines#The_pkgver.28.29_function
Although we use a slightly different, check: https://github.com/BlackArch/blackarch/blob/master/packages/adfind/PKGBUILD#L17
The same is applied to all PKGBUILD files.

Reply:
Perfect, added to all PKGBUILD files and updated the versions to match